challenge.md

```diff
@@ -4,21 +4,21 @@
 
 Write your answers inside this file, where it's indicated by the comments.
 
-1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
+1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that? git clone
 
-2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
+2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit? git diff
```

Comment:
We were looking for `git status`, but based on how this question was worded, `git diff` is a reasonable answer, and I won't take any points off. In the real world, you'll probably be using both `git status` *and* `git diff` to inventory the changes you've made.